lib/Cake/Cache/Cache.php

```diff
@@ -495,6 +495,9 @@ public static function clearGroup($group, $config = 'default') {
  */
 	public static function isInitialized($config = 'default') {
 		if (Configure::read('Cache.disable')) {
+			if (Configure::read('debug') > 1) {
```

Comment:
Yes, I wasn't sure. I was about to do that then thought that it was Cake 3. I'll update this too.